standalone-metastore/metastore-common/src/main/java/org/apache/hadoop/hive/metastore/IMetaStoreClient.java

```diff
@@ -4062,4 +4062,8 @@ void createOrDropTriggerToPoolMapping(String resourcePlanName, String triggerNam
 
   ReplicationMetricList getReplicationMetrics(GetReplicationMetricsRequest
                                                 replicationMetricsRequest) throws MetaException, TException;
+  void setValidWriteIdList(String txnWriteIdList);
```

Comment:
can you rename the variable to validWriteIdList since that is more readable in my opinion.
Also, I think it would be good to document this method especially since this interface is a public API and used by non-hive HMS clients. Do we plan to use this validWriteIdList to set in the HMS API requests if the input arguments don't explicitly provide it?

Reply:
Sure. Yes, that's correct.